pkg/agent/eventsync.go

```diff
@@ -35,7 +35,8 @@ func NewEventSync() (*EventSync, error) {
 		// add attribute migration to migrations
 		attributeMigration := migrate.NewAttributeMigration(agent.apicClient, agent.cfg)
 		ardMigration := migrate.NewArdMigration(agent.apicClient, agent.cfg)
```

Comment:
if we see this getting larger as more migrations are needed, maybe rethink this to another func or something... just TOL